apps/client/src/app/components/rule/rule-settings-dialog/rule-settings-dialog.html

```diff
@@ -1,32 +1,37 @@
 <div mat-dialog-title>{{ data.rule.name }}</div>
 
 <div class="py-3" mat-dialog-content>
-  <mat-form-field
+  <div
     appearance="outline"
     class="w-100"
     [ngClass]="{ 'd-none': !data.rule.configuration.thresholdMin }"
   >
-    <mat-label i18n>Threshold Min</mat-label>
-    <input
-      matInput
+    <label i18n>Threshold Min</label>
+    <mat-slider
       name="thresholdMin"
-      type="number"
-      [(ngModel)]="data.settings.thresholdMin"
-    />
-  </mat-form-field>
-  <mat-form-field
+      [max]="data.rule.configuration.threshold.max"
+      [min]="data.rule.configuration.threshold.min"
+      [step]="data.rule.configuration.threshold.step"
+    >
+      <input matSliderThumb [(ngModel)]="data.settings.thresholdMin" />
+    </mat-slider>
+    <mat-label>{{ data.settings.thresholdMin }}</mat-label>
```

Comment:
```
<label>{{ data.rule.configuration.threshold.max }}</label>
```

Reply:
please correct me if I'm wrong: just to add the label but don't change the value to `data.rule.configuration.threshold.max`